shared/constants/metametrics.js

```diff
@@ -356,15 +356,13 @@ export const EVENT_NAMES = {
   ONBOARDING_WALLET_SECURITY_SKIP_INITIATED: 'SRP Skip Backup Selected',
   ONBOARDING_WALLET_SECURITY_SKIP_CONFIRMED: 'SRP Backup Skipped',
   ONBOARDING_WALLET_SECURITY_SKIP_CANCELED: 'SRP Skip Backup Canceled',
-  ONBOARDING_WALLET_SECURITY_PHRASE_REVEALED: 'Key Material Revealed',
+  ONBOARDING_WALLET_SECURITY_PHRASE_REVEALED: 'SRP Revealed',
   ONBOARDING_WALLET_SECURITY_PHRASE_WRITTEN_DOWN: 'SRP Backup Confirm Display',
   ONBOARDING_WALLET_SECURITY_PHRASE_CONFIRMED: 'SRP Backup Confirmed',
   ONBOARDING_WALLET_CREATION_COMPLETE: 'Wallet Created',
-  ONBOARDING_WALLET_IMPORT_COMPLETE: 'Wallet Imported',
```

Comment:
As per the [issue description](https://github.com/MetaMask/metamask-extension/issues/17669), this needs to be changed to 
`ONBOARDING_WALLET_IMPORT_COMPLETE: 'Wallet Created',`. Why are we removing it here?

Reply:
I removed it an used `ONBOARDING_WALLET_CREATION_COMPLETE` instead, since its value is `'Wallet Created'`